packages/react/src/components/Footer/LegalNav.js

```diff
@@ -12,6 +12,20 @@ import { Link } from 'carbon-components-react';
 
 const { prefix } = settings;
 
+/**
+ * Placeholder <li/> element for injection of the TrustE cookie preferences link
+ *
+ * @returns {*} JSX object
+ */
+const renderTrusteItem = () => {
+  return (
+    <li
+      className={`${prefix}--legal-nav__list-item`}
+      data-autoid="dds-privacy-cp"
```

Comment:
> **From @jeffchew via Slack**
> one thing I'm wondering though is the styling for the link, right now we have the links use `bx--link`, we should probably apply the styling for Juan when they add the link...was thinking possibly setting it in our styles then do an `@extend` of `bx--link` when targeting the `a` element under the nav link item

I'm not crazy about using extend in this scenario or styling the `a` directly. I see no reason why they couldn't inject the link with the class name. They have all of the carbon components, right? Worst case scenario is maybe we can add a utility class where they provide the `url`, and `text`, and we inject it for them?